### PR TITLE
Use Base58 CID's for CAS Interface

### PR DESCRIPTION
**Pull request by BenjaminMoe**
This pull request contains the following changes.

1. Added CAS tests for ion vectors
2. Changed Cas module to return ipfs-format base58 CID's
3. Added force initialization to MockCas to replicate compatibility
4. Updated code where MockCas interface was not being initialized
5. Cleaned up base64/58 conversion functions from Encoder

Test run:
- Ganache local read/write
- Ropsten local read
- Ganache multiple nodes read/write
- AWS S3 cas module
- AWS QLDB ledger module 

Transaction Number: 289
Screenshot from 2022-01-06 03-40-02

Note that the comments marked as hidden on this PR are me taking notes for what needs to be tested and confirmed before marking the PR as ready for review.

### CODE REVIEW

**Comment by BenjaminMoe** [comment]
This last commit patched the places where base58 and base64 are conflicting. 

First point is the core index file url that will be a base58 string
1. https://github.com/transmute-industries/sidetree.js/blob/fix/cas-tests/packages/ledger-ethereum/src/EthereumLedger.ts#L203

Second is this test which specifically provides base64 strings to the above function, which is why the `if` statement is there
2. https://github.com/transmute-industries/sidetree.js/blob/fix/cas-tests/packages/ledger-ethereum/src/__tests__/EthereumLedger.test.ts#L103

Third is the point of reading from the ledger where it looks like a base64 string is being built from a buffer
3. https://github.com/transmute-industries/sidetree.js/blob/fix/cas-tests/packages/ledger-ethereum/src/utils.ts#L34

And then the contract itself only expects to see 32 bytes
https://github.com/transmute-industries/sidetree.js/blob/fix/cas-tests/packages/ledger-ethereum/contracts/SimpleSidetreeAnchor.sol#L21

I think that encoding and decoding to a hash is a preferable option over strings, so i'm going to try doing that on the read side of things and see how it works out.

**Comment by BenjaminMoe** [comment]
In this last commit, I was able to change the anchor string to use the content id of the coreIndexFile directly instead of a base64 representation of the same hash. The next change I want to make is to change this behavior to throw an error when it sees a base64 anchor string, so I can update any fixtures using the wrong format. 

The next two things that I want to make sure are working before I merge this PR, is
1. I want to look at the index files to make sure they look like what ion has
2. I want to run a ganache and ropsten test server to make sure nothing is breaking in practical application.

**Comment by BenjaminMoe** [comment]
In this commit, I removed the base64 to base58 conversion for the anchor string to see which errors were thrown. And it looks like it was a single test. That was updated, and everything should be consistent.

In terms of the anchor string, not much should change because the contract is 32 bytes, so it's just a matter of how we represent that in code. Otherwise the only difference is going to be how CID's are stored in coreIndex and chunk files, and since we added in some compatibility for that, we should still be able to read did's that were written before this change.

The next two things I want to follow up on are:
1. Do the AWS tests still work?
2. Does the dashboard still work?

For the dashboard i might need to be pretty careful. I don't want to add in any data that would cause errors on existing nodes. So I'll test writing and reading with ganache, and then reading with ropsten. Then we can look at the code to try and determine if using the direct base58 CID's would cause any issues.

**Comment by BenjaminMoe** [comment]
Tested dashboard with Ganache. Need to test reading with Ropsten next. And I should probably add a `console.log` into the cas interface to check what content is being returned for a given CID. 

And then I need to find my AWS credentials to make sure that is still working.

**Comment by BenjaminMoe** [comment]
Read-side on Ropsten seems to be working fine. I don't think there's anything that would really cause this to break. In terms of the contract, what's actually being stored is the 32 bytes for the hash of the IPFS CID. So base64 or base58 is only a string representation. So it's only a matter of how we interpret these strings.

The main difference, is that we're writing base58 CID's to the files being stored in IPFS instead of base64. But this should be okay as well as that's how the IPFS-CAS module already works.

https://github.com/transmute-industries/sidetree.js/blob/main/packages/cas-ipfs/src/IpfsCas.ts#L71
```
  public async read(address: string): Promise<FetchResult> {
    if (Encoder.isBase64UrlString(address) && address.indexOf('Ei') === 0) {
      address = Encoder.formatIpfsAddress(address);
    }
```

If a base64 string is provided it will be converted to a base58 CID, otherwise it will be assumed to be a base58 CID. So, nothing should break with existing clients if we switch over to using base58 in the files, because it's already assumed that's the default. So I think this looks good. The only other things is to enable and run the AWS tests locally.

**Comment by BenjaminMoe** [comment]
`cas-s3` is passing all of the tests.

```
> @sidetree/cas-s3@0.3.1-unstable.25 test .../sidetree.js/packages/cas-s3
> tsdx test --passWithNoTests

 PASS  src/S3Cas.spec.ts (37.786s)
  S3Cas
    getServiceVersion
      ✓ should get service version (5ms)
    write
      ✓ should provide an expected cid for a buffer (788ms)
      ✓ should provide an expected cid for a string (779ms)
      ✓ should provide an expected cid for a delta object (755ms)
      ✓ should not match cid with incorrect JSON string (802ms)
    read
      ✓ should produce correct buffer from cid (839ms)
      ✓ should produce correct string from cid (773ms)
      ✓ should produce correct delta object from cid (758ms)
      ✓ should return not found if hash does not exist (775ms)
    files
      ✓ should write and read a createChunkFile file (1559ms)
      ✓ should write and read a createProvisionalIndexFile file (1620ms)
      ✓ should write and read a createCoreIndexFile file (1573ms)
      ✓ should write and read a recoverOperation file (1687ms)
      ✓ should write and read a recoverChunkFile file (1583ms)
      ✓ should write and read a recoverProvisionalIndexFile file (1491ms)
      ✓ should write and read a recoverCoreIndexFile file (1650ms)
      ✓ should write and read a updateCoreProofFile file (1727ms)
      ✓ should write and read a updateChunkFile file (1685ms)
      ✓ should write and read a updateProvisionalIndexFile file (1690ms)
      ✓ should write and read a updateCoreIndexFile file (1540ms)
      ✓ should write and read a deactivateCoreProofFile file (1573ms)
      ✓ should write and read a deactivateCoreIndexFile file (1589ms)
    compatibility
      ✓ should produce correct buffer from hash (770ms)
      ✓ should produce correct string from hash (756ms)
      ✓ should produce correct delta object from hash (762ms)

Test Suites: 1 passed, 1 total
Tests:       25 passed, 25 total
Snapshots:   0 total
Time:        37.854s
Ran all test suites.
```

**Comment by BenjaminMoe** [comment]
QLDB seems to be working as well. 

```
 PASS  src/__tests__/QLDBLedger.test.ts (43.017s)
  QLDB tests
    ✓ constructs object (2433ms)
    ✓ gets service version (1ms)
    ✓ writes to the ledger (7046ms)
    ✓ reads from ledger (2406ms)
    ✓ reads next transaction that got wrote (7072ms)
    ✓ gets multiple transactions (4662ms)
    ✓ should return no transaction if the requested transactionNumber has not been reached (2301ms)
    ✓ should return no transaction if the requested transactionTimeHash doesnt exist (2275ms)

Test Suites: 1 passed, 1 total
Tests:       8 passed, 8 total
Snapshots:   0 total
Time:        43.432s
Ran all test suites.
```

Okay, I think that's all of my concerns addressed. Only other thing to do would be to do a version compatibility test. To see if I can write a did on a node running the updated CAS version, and then read it on an node running the existing version and vise-versa. I can't exactly do this on the exiting testnet, because we need to treat it like a production environment.

What I might try to do instead is see if I can get two localnodes to run from the same ganache-cli, and see if that works before updating this PR from a draft to ready for review.

**Comment by BenjaminMoe** [comment]
Setup a Ganache test that used a single instance of Ganache with a current version node, and a node on this branch and was able to read and write from both nodes using did's from the other node.

File: packages/cas-s3/src/S3Cas.ts [hunk 1]
```diff
@@ -98,6 +97,10 @@ export default class S3Cas implements ICasService {
   }
 
   public async read(address: string): Promise<FetchResult> {
+    if (Encoder.isBase64UrlString(address) && address.indexOf('Ei') === 0) {
```
Comment (OR13) [inline packages/cas-s3/src/S3Cas.ts:100]: do we expect this to be used for ION, ELEM or PHOTON? if no, suggest we remove it.

Comment (BenjaminMoe) [reply packages/cas-s3/src/S3Cas.ts:100]: this is for supporting the exiting base64 CID's that have been written in core Index files.

Comment (OR13) [reply packages/cas-s3/src/S3Cas.ts:100]: I get that, but are those files all just wrong? if they are wrong, let's ship a breaking change and fix it.... they are wrong if they won't ever be used again, and that format won't ever be used in the future... I think this code is preserving a dead branch created by a buggy implementation... I'd rather ship a breaking change now that later.

Comment (BenjaminMoe) [reply packages/cas-s3/src/S3Cas.ts:100]: Sure, I'll take that out the base64 compatibility and then update the PR.

**Comment by BenjaminMoe** [comment]
In the case we just take out the base64 part, the application doesn't fail gracefully. So I'll look into adding some error handling for malformed core index files, probably need to be there anyways. 

Screenshot from 2022-01-04 05-07-29

**Comment by BenjaminMoe** [comment]
Since I need to make more fixes, I'll probably use this PR to make notes for what needs to be done. Close it, and then open a new PR. My thinking was to add a shim to check for a base64 CID, and then convert back to base58 to be able to read from IPFS. I think orie made the correct call in not including this functionality. 

My reasoning for thinking this was is that if we need to make adjustments going forward, I'd rather be able to adapt to new changes, and not try to keep compatibility for a bug from pre-release. How ever, this means that we actually have a harder task on hands, in that we need to make sure that Element "fails gracefully" when we find a malformed core-index file, which is something that might happen. We also might need to set some kind of starting block height as well to clear out the pre-release data.

In terms of response, it seems like a good opportunity to get in some practice for handling this stuff now, because we can't be sure that everything is always going to work perfectly.

**Comment by BenjaminMoe** [comment]
I was thinking that I would update the interface locally to use base58 and then do testing to figure out what needs to be done for that to work, but since we're going forward on this, I might as well commit and then make a list of things that break, and then approach each one of them.

**Comment by BenjaminMoe** [comment]
Locally it looks like I am able to read and write with Ganche. When I try to do reads on Ropsten I get 

```
Invalid core file found for anchor string '1.QmPb14jRhzmwsCRnf9F3oeikHPS6f3BZVyagn25HLe4jAy', the entire batch is discarded. Error: core_index_file_decompression_failure: incorrect header check
Removing transaction '105' from unresolvable transactions if exists...
Finished processing transaction '105'.
Invalid core file found for anchor string '3.QmVPiWn2hLQ13rhz5yCHTwoNYMVPBdV28Rzmgkz3jcwL5H', the entire batch is discarded. Error: core_index_file_decompression_failure: incorrect header check
```

**Comment by BenjaminMoe** [comment]
I think i've confirmed that Element seems to already have error handling working. We are able to decompress anchor files and read the contents.

```
--- CORE INDEX URI ---
QmcxBQceZNKa4cmqZNFKFRqrBc5p7JymJ3L39Z8A3iN3Gf
--- 02 CORE INDEX FILE ---
CoreIndexFile {
  model: {
    provisionalIndexFileUri: 'EiC9Zy4VpPLmvlmHlp0bW8lpS0Bmy4qZwWlc8gd1dhV2KQ',
    operations: { create: [Array] }
  },
  didUniqueSuffixes: [ 'EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A' ],
  createDidSuffixes: [ 'EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A' ],
  recoverDidSuffixes: [],
  deactivateDidSuffixes: []
}
```
When the invalid base64 content id is read from ipfs, we get a content not found error. 
```
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"content_not_found"}
Event emitted: sidetree_download_manager_download: {"code":"success"}
```

Which again is what we want, but it looks like it will attempt to retry.
```
Processed 1 operations. Retry needed: true
Recording failed processing attempt for transaction '276'...
Processed 1 operations. Retry needed: true
Recording failed processing attempt for transaction '282'...
Processed 1 operations. Retry needed: true
Recording failed processing attempt for transaction '285'...
Processed 1 operations. Retry needed: true
Recording failed processing attempt for transaction '279'...
```

So I'm not sure how many times we want to retry until we give up. I think that can be a separate ticket if needed. Or create an issue for it? But the good news is that it looks like there is already base58 data that already exists in the contract that we are able to read. Which means it's probably okay to try a test write.

**Comment by BenjaminMoe** [comment]
Okay, it just looks like it will continue to loop over the most recent transaction, which in this case is 288. But it does that anyways. So i think this is behavior that already exists. The important part is our updated node isn't attempting to retry the previous transactions over and over again, it's only looping over the most recent transaction.

**Comment by BenjaminMoe** [comment]
Did a test write with `did:elem:ropsten:EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A`, looks like we are able to resolve it.

**Comment by BenjaminMoe** [comment]
We might have a problem? 

```
Skipped bad operation for DID EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A at time 11611494. Error: {"stack":"Error: create_operation_missing_or_unknown_property\n    at Function.parseObject (webpack-internal:///../core/dist/core.esm.js:2169:13)\n    at _callee$ (webpack-internal:///../core/dist/core.esm.js:2137:49)\n    at tryCatch (webpack-internal:///../core/dist/core.esm.js:216:40)\n    at Generator.invoke [as _invoke] (webpack-internal:///../core/dist/core.esm.js:447:22)\n    at Generator.eval [as next] (webpack-internal:///../core/dist/core.esm.js:272:21)\n    at asyncGeneratorStep (webpack-internal:///../core/dist/core.esm.js:62:24)\n    at _next (webpack-internal:///../core/dist/core.esm.js:84:9)","message":"create_operation_missing_or_unknown_property","code":"create_operation_missing_or_unknown_property"}
Skipped bad operation for DID EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A at time 11611565. Error: {"stack":"Error: create_operation_missing_or_unknown_property\n    at Function.parseObject (webpack-internal:///../core/dist/core.esm.js:2169:13)\n    at _callee$ (webpack-internal:///../core/dist/core.esm.js:2137:49)\n    at tryCatch (webpack-internal:///../core/dist/core.esm.js:216:40)\n    at Generator.invoke [as _invoke] (webpack-internal:///../core/dist/core.esm.js:447:22)\n    at Generator.eval [as next] (webpack-internal:///../core/dist/core.esm.js:272:21)\n    at asyncGeneratorStep (webpack-internal:///../core/dist/core.esm.js:62:24)\n    at _next (webpack-internal:///../core/dist/core.esm.js:84:9)\n    at runNextTicks (internal/process/task_queues.js:60:5)\n    at processImmediate (internal/timers.js:437:9)","message":"create_operation_missing_or_unknown_property","code":"create_operation_missing_or_unknown_property"}
Skipped bad operation for DID EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A at time 11611994. Error: {"stack":"Error: create_operation_missing_or_unknown_property\n    at Function.parseObject (webpack-internal:///../core/dist/core.esm.js:2169:13)\n    at _callee$ (webpack-internal:///../core/dist/core.esm.js:2137:49)\n    at tryCatch (webpack-internal:///../core/dist/core.esm.js:216:40)\n    at Generator.invoke [as _invoke] (webpack-internal:///../core/dist/core.esm.js:447:22)\n    at Generator.eval [as next] (webpack-internal:///../core/dist/core.esm.js:272:21)\n    at asyncGeneratorStep (webpack-internal:///../core/dist/core.esm.js:62:24)\n    at _next (webpack-internal:///../core/dist/core.esm.js:84:9)","message":"create_operation_missing_or_unknown_property","code":"create_operation_missing_or_unknown_property"}
Skipped bad operation for DID EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A at time 11612000. Error: {"stack":"Error: create_operation_missing_or_unknown_property\n    at Function.parseObject (webpack-internal:///../core/dist/core.esm.js:2169:13)\n    at _callee$ (webpack-internal:///../core/dist/core.esm.js:2137:49)\n    at tryCatch (webpack-internal:///../core/dist/core.esm.js:216:40)\n    at Generator.invoke [as _invoke] (webpack-internal:///../core/dist/core.esm.js:447:22)\n    at Generator.eval [as next] (webpack-internal:///../core/dist/core.esm.js:272:21)\n    at asyncGeneratorStep (webpack-internal:///../core/dist/core.esm.js:62:24)\n    at _next (webpack-internal:///../core/dist/core.esm.js:84:9)","message":"create_operation_missing_or_unknown_property","code":"create_operation_missing_or_unknown_property"}
Skipped bad operation for DID EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A at time 11612061. Error: {"stack":"Error: create_operation_missing_or_unknown_property\n    at Function.parseObject (webpack-internal:///../core/dist/core.esm.js:2169:13)\n    at _callee$ (webpack-internal:///../core/dist/core.esm.js:2137:49)\n    at tryCatch (webpack-internal:///../core/dist/core.esm.js:216:40)\n    at Generator.invoke [as _invoke] (webpack-internal:///../core/dist/core.esm.js:447:22)\n    at Generator.eval [as next] (webpack-internal:///../core/dist/core.esm.js:272:21)\n    at asyncGeneratorStep (webpack-internal:///../core/dist/core.esm.js:62:24)\n    at _next (webpack-internal:///../core/dist/core.esm.js:84:9)","message":"create_operation_missing_or_unknown_property","code":"create_operation_missing_or_unknown_property"}
Skipped bad operation for DID EiCtwD11AV9e1oISQRHnMJsBC3OBdYDmx8xeKeASrKaw6A at time 11612068. Error: {"stack":"Error: create_operation_missing_or_unknown_property\n    at Function.parseObject (webpack-internal:///../core/dist/core.esm.js:2169:13)\n    at _callee$ (webpack-internal:///../core/dist/core.esm.js:2137:49)\n    at tryCatch (webpack-internal:///../core/dist/core.esm.js:216:40)\n    at Generator.invoke [as _invoke] (webpack-internal:///../core/dist/core.esm.js:447:22)\n    at Generator.eval [as next] (webpack-internal:///../core/dist/core.esm.js:272:21)\n    at asyncGeneratorStep (webpack-internal:///../core/dist/core.esm.js:62:24)\n    at _next (webpack-internal:///../core/dist/core.esm.js:84:9)","message":"create_operation_missing_or_unknown_property","code":"create_operation_missing_or_unknown_property"}
```

File: packages/cas/src/tests/__fixtures__/index.ts [hunk 2]
```diff
@@ -17,146 +17,156 @@
...
   },
   createCoreIndexFile: {
     cid: 'QmYtZhCUqbyvuQrjNHRtYvh2NRtog1xPfxsqC3AXDK8oen',
-    data: Buffer.from(
+    content: Buffer.from(
       '1f8b080000000000000325cdc16e82300080e177e979878103851ba0885388202cb065592ab4a50c282b05bb11de7d18cf7ff2fd13e8381b694f590beb435b20e9d21a259c0213844d70239acf2bb7dc17672c7efcd8cea8c661160ca5a2aadb6acc3c2b4c043f4a3d044f80758843b1503d30279073040502e6c704fa01632ab750c07b28502da007fb7279eca8a39cf6b87c59bfdb69e546e9ee2458fbbae99c2bd173c3be0a2fb588a56ae3f91293e5c151ce46c47f1dd6345434a8150f65e57fa9c5e12db1d9b0d213031f633bd0fd3f435aaa9451a490defb769ed71be37203f3fc39cfff5b850a70fa000000',
       'hex'
     ),
-    json:
+    jsonStr:
```
Comment (OR13) [inline packages/cas/src/tests/__fixtures__/index.ts:51]: I love this name change. much clearer.

File: packages/cas/src/tests/__fixtures__/index.ts [hunk 3]
```diff
@@ -17,146 +17,156 @@
...
       '{"provisionalIndexFileUri":"QmNwg5MrjFhGdPftqMTBYi5raYNuh122DjvYHAQUtrKx6Q","operations":{"create":[{"suffixData":{"deltaHash":"EiC1LGfh47ZBXjFRXELtonJ8pCbg6c9BbtHXAgA25vPSTg","recoveryCommitment":"EiC3M_2dIVUBou36U9fKTBN6Mz9xA2xxRR1gsHkC0789Sw"}}]}}',
   },
   recoverOperation: {
     cid: 'QmWA2bmcdTLAVDMjddeFJvw4fdxoNkJSUvUhKJiipPgP7R',
-    data: Buffer.from(
+    content: Buffer.from(
```
Comment (OR13) [inline packages/cas/src/tests/__fixtures__/index.ts:56]: another good name change, well done.

File: packages/cas/src/tests/__fixtures__/index.ts [hunk 4]
```diff
@@ -17,146 +17,156 @@
...
+ * https://github.com/decentralized-identity/sidetree/blob/master/tests/core/BatchWriter.spec.ts
+ *
+ * Test Buffer was taken from test:
+ * 'should return without writing anything if operation queue is empty.'
+ *
+ * Test String was taken from test:
+ * 'should pass the writer lock ID to CoreIndexFile.createBuffer() if a value lock exists.'
+ *
+ * By Logging the values passed into the MockCas write function during the test
+ * https://github.com/decentralized-identity/sidetree/blob/master/tests/mocks/MockCas.ts#L34
```
Comment (OR13) [inline packages/cas/src/tests/__fixtures__/index.ts:157]: great comments

File: packages/cas/src/tests/__fixtures__/index.ts [hunk 5]
```diff
@@ -17,146 +17,156 @@
...
 export const testBuffer = Buffer.from(
   '1f8b08000000000000034d91db6ee2301086dfc5d73dc5b4bb2d7729a5a1453408e8525a21e4d893c4cdc1966343bc28efbe362c6d255f8cfff9e7f3cc788f18949a34a8ffb1474632a26120aa8aeb0a6a8dfa68c8c3cdb9685e2c4e696ca2e8f760160f9377f5ccec34bca6365946b3ccacdab45d1457193a4392689ac39147a8e6a2761405b224145c9a096a8ee83d922629391d833dba3973ce866735af33273ab3b6127c0f9435640e54e29b5f45f007144f39251eed6cf82ab8f3cf9e58cfbbc2b30b6d7de5c0a5a8da7af0a9de29adbb7f62dc9b3d9a84f5a2e5e805bfc5f1b99e878b30d4e5fd2bbef91b4c8a16f7623adebcba0a0f63f3227f8b69c03f37ed232b96228f52939adbebc9f41ea623bb9a8c47692b57b1a1a8f31d29299ac32610313a7743ffefdaf148d380f2f104742e98ef924892f0926bfb546fc597f15b7e8012b2935c800d33057058e5ba5b9fb9f9d49653f8b9caa3f2c402dcfb5e66232a58f8f0ab64583329f8e1b773ad65d3bfbcdced76174695175454a85b77eeacbb7fdffa6cbc2a020000',
   'hex'
 );
 export const testString = 'anyCoreIndexFileBuffer';
 
-export const testBufferHash = 'EiAuYMQ4_ZBO1ckbGpyyDKlcq8EFpG3duG3nkhR79RwFQQ';
-export const testStringHash = 'EiBLkBZiYEmzDAD-CuMapW18cBaQr0un-cG0AjiLfdFhZA';
-export const testObjectHash = 'EiAxghX7H2qN_c_wwt0GCcCVRt0JEhAPw3UoHS9JS-5cRg';
+// Base 58 (Ipfs Content Id)
```
Comment (OR13) [inline packages/cas/src/tests/__fixtures__/index.ts:166]: technically an IPFS v1 CID.

**Review by OR13 — APPROVED** [review]

**Pull request merged**